rollup/cmd/rollup_relayer/app/app.go

```diff
@@ -96,6 +96,13 @@ func action(ctx *cli.Context) error {
 		log.Crit("cfg.L2Config.ChunkProposerConfig.MaxL2GasPerChunk must be greater than 0")
 	}
 
+	if cfg.L2Config.RelayerConfig.ValidiumMode {
+		// Force single batch submission for validium mode, this looks a bit hacky but it avoids more changes in the relayer code.
```

Comment:
a better change is set the default value as 1, when set it in config, then use the config value